### PR TITLE
Placement: add grpc get table api

### PR DESCRIPTION
**Pull request by Taction**
# Description

Add grpc GetPlacementTables API to get all host infos.

## Issue reference

Please reference the issue this PR will close: #6033

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: dapr/proto/placement/v1/placement.proto [hunk 1]
```diff
@@ -46,3 +51,15 @@ message Host {
   repeated string entities = 4;
   string id = 5;
 }
+
+message HostInfo {
+  string name = 1;
+  string app_id = 2;
+  repeated string entities = 3;
+  int64 update_at = 4;
```
Comment (ItalyPaleAle) [inline dapr/proto/placement/v1/placement.proto:59]: `updated_at` or just `updated`

File: pkg/placement/placement.go [hunk 2]
```diff
@@ -181,6 +182,26 @@ func (p *Service) Run(ctx context.Context, port string) error {
...
 }
 
+// GetPlacementTables returns the current placement host infos.
+func (p *Service) GetPlacementTables(ctx context.Context, empty *emptypb.Empty) (*placementv1pb.GetPlacementTablesResponse, error) {
+	m := p.raftNode.FSM().State().Members()
+	version := p.raftNode.FSM().State().TableGeneration()
+	response := &placementv1pb.GetPlacementTablesResponse{
+		TableVersion: version,
+	}
+	members := make(map[string]*placementv1pb.HostInfo)
```
Comment (ItalyPaleAle) [inline pkg/placement/placement.go:192]:
```suggestion
	members := make(map[string]*placementv1pb.HostInfo, len(m))
```

File: dapr/proto/placement/v1/placement.proto (hunk 1 above)
Comment (Taction) [reply dapr/proto/placement/v1/placement.proto]: Thanks, changed to updated_at.

File: pkg/placement/placement.go (hunk 2 above)
Comment (Taction) [reply pkg/placement/placement.go]: Done.

**Review by ItalyPaleAle — APPROVED** [review]

File: pkg/placement/placement.go [hunk 3]
```diff
@@ -241,11 +262,12 @@ func (p *Service) ReportDaprStatus(stream placementv1pb.Placement_ReportDaprStat
 						UpdatedAt: p.clock.Now().UnixNano(),
 					},
 				}
+				log.Debugf("Member changed upserting appid %s with entities %v", req.Id, req.Entities)
```
Comment (mukundansundar) [inline pkg/placement/placement.go:265]: I think this debug log is not necessary.

**Review by yaron2 — APPROVED** [review]

**Pull request merged**